HW_3/models.py

```diff
@@ -0,0 +1,12 @@
+from flask_sqlalchemy import SQLAlchemy 
+
+
+db = SQLAlchemy()
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_name = db.Column(db.String(80), nullable=False)
+    user_surname = db.Column(db.String(120), nullable=False)
+    email = db.Column(db.String(120), unique=True, nullable=False)
+    password = db.Column(db.String(256), nullable=False)
```

Comment:
Использование функции `sha256` и метода `hexdigest` даст хэш из 64 символов. Такое длинное поле не нужно.
```suggestion
    password = db.Column(db.String(64), nullable=False)
```